cmake-tool/simulate_scripts/simulate.py

```diff
@@ -80,23 +80,25 @@ def notice(message):
                                   qemu_gdbserver_command]
     qemu_simulate_command = " ".join(qemu_simulate_command_opts)
 
-    notice(qemu_simulate_command)
-
     if qemu_gdbserver_command != "":
-        notice('waiting for GDB on port 1234...')
-
-    qemu_status = subprocess.call(qemu_simulate_command, shell=True)
-
-    if qemu_status != 0:
-        delay = 5  # in seconds
-        # Force a newline onto the output stream.
-        sys.stderr.write('\n')
-        msg = "QEMU failed; resetting terminal in {d} seconds".format(d=delay) \
-            + "--interrupt to abort\n"
-        notice(msg)
-    else:
-        delay = 2  # in seconds
-
-    time.sleep(delay)
-
-    subprocess.call("tput reset", shell=True)
+        print(f'GDB command: {qemu_gdbserver_command}', file=sys.stderr)
```

Comment:
Shouldn't this be printed only if `args.dry_run` is set?

Reply:
Yes, this is a bit unfortunate sometimes, but it survives in captured logs at least.
But I think this line is meant to be printed to give the user a hint what to do next, so it should be printed always. Thus Ignore my remark.